examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
I am down for `persist-dismissal="false"` naming. In #2118 @rudygalfi originally suggested `forget-dismissal` but I think @dvoytenko and @cramforce liked `always-show` better. Dima, Malte, should we go with a different naming?